scripts/ch-trace-visualiser.py

```diff
@@ -0,0 +1,98 @@
+#!/bin/env python3
+#
+# Copyright © 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from colorsys import hsv_to_rgb
+from random import random
+import xml
+import json
+from sys import argv, stderr
+import xml.etree.ElementTree as ET
```

Comment:
Is there any Python lib that needs to be installed manually for this script to be supported?

Reply:
Nope! I was careful to only use modules from the standard library: https://docs.python.org/3/library/